packages/transitive-overlay/src/util.ts

```diff
@@ -0,0 +1,276 @@
+import {
+  Company,
+  Itinerary,
+  Leg,
+  TransitiveData
+} from "@opentripplanner/types";
+import coreUtils from "@opentripplanner/core-utils";
+import { getPlaceName } from "@opentripplanner/itinerary-body/lib/util";
```

Comment:
This is a new dependency to the itinerary-body package. Please move that dependency in the `package.json` for the `transitive-overlay` package from a dev to a full dependency.